### PR TITLE
RD-2593 Migrate BlueprintActionButtons widget to TS

### PR DESCRIPTION
**Pull request by Gelio**
This PR migrates the Blueprint Action Buttons widget to TypeScript. There shouldn't be any functional changes in this PR.

I have decided to separate these 2 commits to another PR in order to make the reverting process easier in 6.2, when the button should be shown. By having the PRs for RD-2593 separated, we could only revert the 2nd PR (the one that hides the "Edit a copy in Composer" button) and not revert this TS migration.

## System tests

https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/856/pipeline

Queued 2021-06-28 14:55 CEST

### CODE REVIEW

File: widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+
+export default class BlueprintActionButtons extends React.Component<
+    BlueprintActionButtonsProps,
+    BlueprintActionButtonsState
+> {
+    // eslint-disable-next-line react/static-property-placement
+    static propTypes = {
+        blueprintId: PropTypes.string,
+        toolbox: Stage.PropTypes.Toolbox.isRequired
+    };
```
Comment (kubama) [inline widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx:24]: Do we still need this? If yes, why it was moved from outside of the class to here? Also, why do we need eslint disable?

**Review by kubama — COMMENTED** [review]
Looks good!
Did you consider converting `BlueprintActionButtons` to functional component at the same time?

**Comment by Gelio** [comment]
> Did you consider converting BlueprintActionButtons to functional component at the same time?

Nope :smile: These conversions are usually not diff-friendly, so I decided not to do it this time and finish the scope earlier :slightly_smiling_face:

File: widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx (hunk 1 above)
Comment (Gelio) [reply widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx]: Done, I've removed the `propTypes` :slightly_smiling_face:

**Review by kubama — APPROVED** [review]

**Review by stepek — APPROVED** [review]
Lgtm!

**Pull request merged**